sos/policies/__init__.py

```diff
@@ -190,10 +190,17 @@ def fmt_container_cmd(self, container, cmd):
         :param cmd: The command to run inside `container`
         :type cmd: ``str``
 
+        :param quote: Whether the cmd should be quoted.
+        :type quote: ``bool``
+
         :returns: Formatted string to run `cmd` inside `container`
         :rtype: ``str``
         """
-        return "%s %s %s" % (self.run_cmd, container, quote(cmd))
+        if quote:
+            quoted_cmd = quote(cmd)
+        else:
+            quoted_cmd = cmd
+        return "%s %s %s" % (self.run_cmd, container, quoted_cmd)
```

Comment:
There's a potential of a scoping issue (and slightly readability to) by using `quote` as both a variable and a method. It's fine if the parameter to `Plugin.fmt_container_runtime()` is `quote`, but for `ContainerRuntime.fmt_container_runtime()` let's name it something else to avoid that issue. E.G.

```python
    def fmt_container_cmd(self, container, cmd, quotecmd=False):
        """Format a command to run inside a container using the runtime

        :param container: The name or ID of the container in which to run
        :type container: ``str``

        :param cmd: The command to run inside `container`
        :type cmd: ``str``

        :param quotecmd: Should the command be quoted for execution
        :type quotecmd: ``bool``

        :returns: Formatted string to run `cmd` inside `container`
        :rtype: ``str``
        """
        return "%s %s %s" % (self.run_cmd,
                             container,
                             quote(cmd) if quotecmd else cmd)
```